### PR TITLE
v1.2.0 – Fixing up for gh-pages branch

### PR DESCRIPTION
**Pull request by ashleynolan**
### Added
- Dangerfile and travis build configs added

### Fixed
- Fixed baseUrl links for push to `gh-pages`
- Fixed link to radio and checkbox partials

### CODE REVIEW

File: dangerfile.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    // Fail if the title of the PR isn't in the format of a version i.e. vX.X.X (such as v1.4.0)
+    const versionRegex = /^(v[0-9]+\.[0-9]+\.[0-9]+)/;
+    const isPRTitleVersioned = (danger.github.pr.title).match(versionRegex);
+    if (!isPRTitleVersioned) {
+        fail(`:exclamation: PR title should start with the package version in the format v(x.x.x) (such as v1.4.0)`);
+    }
+
+    // Fail if there isn’t a CHANGELOG entry – should update for every PR
+    if (!danger.git.modified_files.includes('CHANGELOG.md')) {
+        const changelogLink = 'https://github.com/justeat/fozzie/blob/master/CHANGELOG.md';
```
Comment (80sgreek) [inline dangerfile.js]: Is this the correct changelog location?

File: docs/src/templates/pages/index.hbs [hunk 2]
```diff
@@ -35,7 +35,7 @@ This documentation is evolving over time, so if something is missing that you wo
         	<li class="g-col g-span6--narrow g-span4--mid list--tiled-item list--tiled-item--dark">
```
Comment (80sgreek) [inline docs/src/templates/pages/index.hbs]: Consistent indentation would be nice but not that much of a biggy

File: dangerfile.js (hunk 1 above)
Comment (ashleynolan) [reply dangerfile.js]: Updated 🔍

File: docs/src/templates/pages/index.hbs (hunk 2 above)
Comment (ashleynolan) [reply docs/src/templates/pages/index.hbs]: Yep - have changed

**Comment by ashleynolan** [comment]
@kevinrodrigues @80sgreek Any chance you could take a look again now I've got the build green and addressed the comments?

**Pull request merged**